README.md

```diff
@@ -41,13 +41,32 @@ Formats the token amount.
 #### Parameters
 
 - `options.symbol`: the token symbol, as a `String`. Overrides the value set in the constructor.
+- `options.commify`: whether the token amount should have comma separators
 - `options.digits`: the number of digits to display. Defaults to `2`.
 - `options.displaySign`: whether the sign (`-` or `+`) should be displayed for the amount.
 
 ### TokenAmount#format(options)
 
 Alias to TokenAmount#format().
 
+### TokenAmount#convert(rate, decimals, options)
+
+Converts from a rate, returning a new `TokenAmount` instance with the desired decimals and set
+options.
```

Comment:
Note that we generally don't linebreak in markdown files 😄

Reply:
Haha my editor linebreaking!